assets/scss/generic-overrides.scss

```diff
@@ -27,19 +27,19 @@
 
 //end image extending
 //style default button as secondary button
-
-.wp-block-button__link, .wp-block-file__button, .wp-block-file a.wp-block-file__button, .wp-block-file a.wp-block-file__button:active, .wp-block-file a.wp-block-file__button:focus, .wp-block-file a.wp-block-file__button:hover, .wp-block-file a.wp-block-file__button:visited {
-  @extend .nhsuk-button;
-  border-radius : $nhsuk-border-radius;
-  background    : $nhsuk-button-color;
-  font-size     : $nhsuk-button-text-color;
-  padding       : 1px nhsuk-spacing(2);
-  position      : relative;
-  top           : -nhsuk-spacing(1);
-  margin-bottom : nhsuk-spacing(1);
+#content {
+  a.wp-block-button__link, .wp-block-file__button, .wp-block-file a.wp-block-file__button, .wp-block-file a.wp-block-file__button:active, .wp-block-file a.wp-block-file__button:focus, .wp-block-file a.wp-block-file__button:hover, .wp-block-file a.wp-block-file__button:visited {
+	@extend .nhsuk-button;
+	border-radius : $nhsuk-border-radius;
+	background    : $nhsuk-button-color;
+	/*padding       : 1px nhsuk-spacing(2);*/
```

Comment:
Should this be removed not commented out?

Reply:
updated to remove this